orchestrator/lib/coolfhir/azure_test.go

```diff
@@ -73,3 +73,59 @@ func Test_azureHttpClient_Do(t *testing.T) {
 		require.Equal(t, "application/fhir+json", capturedHeaders.Get("Content-Type"))
 	})
 }
+
+func Test_NewFHIRAuthRoundTripper(t *testing.T) {
```

Comment:
```suggestion
func Test_NewAuthRoundTripper(t *testing.T) {
```

Reply:
Good catch, I used the IDE to rename so it missed this